openadapt/app/dashboard/components/RecordingDetails/RecordingDetails.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { timeStampToDateString } from '@/app/utils';
+import { Recording } from '@/types/recording'
+import { Table } from '@mantine/core'
+import React from 'react'
+
+type Props = {
+    recording: Recording;
+}
+
+export const RecordingDetails = ({
+    recording
+}: Props) => {
+  return (
```

Comment:
Formatting appears to be inconsistent here. Ideally we should implement `prettier` and have this done automatically -- tracked in https://github.com/OpenAdaptAI/OpenAdapt/issues/636.